geocoder/hierarchy.cpp

```diff
@@ -81,10 +84,19 @@ void Hierarchy::Entry::DeserializeFromJSONImpl(json_t * const root, string const
   else if (m_nameTokens != m_address[static_cast<size_t>(m_type)])
   {
     ++stats.m_mismatchedNames;
-    LOG(LDEBUG, ("Hierarchy entry name is not the most detailed field in its address:", jsonStr));
   }
 }
 
+bool Hierarchy::Entry::IsParentTo(Hierarchy::Entry const & e) const
+{
+  for (size_t i = 0; i < static_cast<size_t>(geocoder::Type::Count); ++i)
```

Comment:
Maybe add constexpr function? Something like `geocoded::get_types_count` ?

Reply:
We don't usually do this.